how-it-works/2_chainkeytech/04_chain_key_tokens.card.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Chain-key Tokens
```

Comment:
```suggestion
title: Chain-Key Tokens
```

Reply:
The card is lowercase, so made now all but the first word lowercase.